packages/web/src/utils/sagaHelpers.js

```diff
@@ -104,6 +105,15 @@ export function* doEvery(millis, fn, times = null) {
   return chan
 }
 
+export function* waitForAccount() {
+  yield call(
+    waitForValue,
+    (state) => state.account.status,
+    null,
+    (status) => status !== Status.LOADING
```

Comment:
Nice

Reply:
yeah this felt good, thank the lord, lol